File: scripts/Power of Three/Power of Three.js
var isPowerOfThree = function(n) {
    if(n === 1){return true;}
    if(n === 0){return false;}
    
    n/=3;
    
    if(n%3 != 0 && n != 1){
        return false;
    }else{
        let x = isPowerOfThree(n);
        return x;
    }
};

File: scripts/algorithms/Power of Three/Power of Three.java
class Solution {
    public boolean isPowerOfThree(int n) {
       if(n==1){
            return true;
        }
        if(n<=0){
            return false;
        }
        if(n%3 !=0 && n>1){
            return false;
        }
        else{
            return isPowerOfThree(n/3); // recurssion 
        }
    }
}
